backend/uptobox/uptobox.go

```diff
@@ -0,0 +1,1052 @@
+package uptobox
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/rclone/rclone/backend/uptobox/api"
+	"github.com/rclone/rclone/fs"
+	"github.com/rclone/rclone/fs/config"
+	"github.com/rclone/rclone/fs/config/configmap"
+	"github.com/rclone/rclone/fs/config/configstruct"
+	"github.com/rclone/rclone/fs/fserrors"
+	"github.com/rclone/rclone/fs/fshttp"
+	"github.com/rclone/rclone/fs/hash"
+	"github.com/rclone/rclone/lib/encoder"
+	"github.com/rclone/rclone/lib/pacer"
+	"github.com/rclone/rclone/lib/random"
+	"github.com/rclone/rclone/lib/rest"
+)
+
+const (
+	apiBaseURL     = "https://uptobox.com/api"
+	minSleep       = 400 * time.Millisecond // api is extremely rate limited now
+	maxSleep       = 5 * time.Second
+	decayConstant  = 2 // bigger for slower decay, exponential
+	attackConstant = 0 // start with max sleep
+)
+
+func init() {
+	fs.Register(&fs.RegInfo{
+		Name:        "uptobox",
+		Description: "Uptobox",
+		Config: func(ctx context.Context, name string, config configmap.Mapper) {
+		},
+		NewFs: NewFs,
+		Options: []fs.Option{{
+			Help: "Your API Key, get it from https://uptobox.com/my_account",
```

Comment:
```
Your API Key, get it from https://uptobox.com/my_account
Enter a string value. Press Enter for the default ("").
```

The URL printed is correct , but the thing you need to get is called the `token`

![image](https://user-images.githubusercontent.com/536803/114411398-0bcf2a00-9ba4-11eb-82be-a82e308a62ff.png)